Assets/Scenes/Scripts/DestroyByContact.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DestroyByContact : MonoBehaviour {
+	public GameObject explosion;
+
+	public int astroid_health;
+
+	public GameObject healthbar;
+	public Sprite sp_00;
+	public Sprite sp_05;
+
+	void hit(){
+
+		astroid_health--;
```

Comment:
```suggestion
		--astroid_health;
```
Always -- first when working with Colan, jk but we usually do -- first in most repos so it's a good habit to get used to